src/main/java/fruitshop/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,24 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.Storage;
+
+public class BalanceOperation implements OperationHandler {
+    public static final String READ_FILE_PATH = "src/main/resources/reportToRead.csv";
```

Comment:
Avoid hardcoding file paths within classes. It's better to pass the file path as a parameter to the method that requires it. This will make your code more flexible and easier to test.